bench/blob.js

```diff
@@ -0,0 +1,62 @@
+const Benchmark = require('benchmark');
+const suite = new Benchmark.Suite();
+const { eventToMdTable, H2, createTableHeader } = require('../markdown');
+const { Blob } = require('buffer');
+
+const tableHeader = createTableHeader(['name', 'ops/sec', 'samples']);
+
+const blob128 = new Blob(Buffer.allocUnsafe(128));
+const blob1024 = new Blob(Buffer.allocUnsafe(1024));
+
+const options128 = { defer: true };
+const options1024 = { defer: true };
+
+suite
+  .add(
```

Comment:
can you also add the `new Blob` creation to benchmark?

Reply:
Done